src/Components/Login/Login.jsx

```diff
@@ -1,23 +1,13 @@
 /* eslint-disable import/no-named-as-default, no-shadow, consistent-return */
-import React, { Component } from 'react';
+import React from 'react';
 import { Grid, Card, Button } from 'semantic-ui-react';
-// import { connect } from 'react-redux';
-// import PropTypes from 'prop-types';
+
 import './Login.css';
 
-// NOTE all commented out code will be used in future mapping of state, dispatching of actions etc
-// import { renderSheets } from '../../actions/renderActions';
-// import injectWidgetId from '../../utils/utils';
-/* eslint-disable react/prefer-stateless-function */
-export class Login extends Component {
-  // componentDidMount() {
-  //   if (cookie.load('userName') !== undefined) {
-  //     const name = cookie.load('userName');
-  //     // NOTE write function to trigger the render of the main sheets page
-  //   }
-  // }
-  render() {
-    const GOOGLE_AUTH_URL = process.env.REACT_APP_GOOGLE_AUTH_URL;
+export const Login = () => {
```

Comment:
@Gwillison415 thanks also for recognizing this is now more well suited as a functional component.